pkg/capabilities/crypto/crypto.go

```diff
@@ -47,23 +46,28 @@ func VerifyCert(h *cap.Host, cert Certificate, certChain []Certificate, notAfter
 
 	payload, err := json.Marshal(requestObj)
 	if err != nil {
-		return false, fmt.Errorf("cannot serialize request object: %w", err)
+		return CertificateVerificationResponse{}, fmt.Errorf("cannot serialize request object: %w", err)
 	}
 
 	// perform callback
 	responsePayload, err := h.Client.HostCall("kubewarden", "crypto", "v1/is_certificate_trusted", payload)
 	if err != nil {
-		return false, err
+		return CertificateVerificationResponse{}, err
 	}
 
 	responseObj := CertificateVerificationResponse{}
 	if err := json.Unmarshal(responsePayload, &responseObj); err != nil {
-		return false, fmt.Errorf("cannot unmarshall response object: %w", err)
+		return CertificateVerificationResponse{}, fmt.Errorf("cannot unmarshall response object: %w", err)
 	}
 
 	if responseObj.Trusted {
-		return true, nil
+		return CertificateVerificationResponse{
+			Trusted: true,
+		}, nil
 	} else {
-		return false, fmt.Errorf(responseObj.Reason)
+		return CertificateVerificationResponse{
+			Trusted: false,
+			Reason:  responseObj.Reason,
+		}, nil
 	}
```

Comment:
This can be deleted, just return the `responseObj` you just unmarshalled

Reply:
indeed, what a brainfart. Fixed.